doc/latex.rst

```diff
@@ -783,6 +783,8 @@ Macros
      multiple paragraphs in header cells of tables.
   .. versionadded:: 1.6.3
      ``\sphinxstylecodecontinued`` and ``\sphinxstylecodecontinues``.
+  .. versionadded:: 3.0.0???
```

Comment:
I think this PR does not change the behavior of Sphinx. So this is not a breaking change. So I prefer to add this as 2.4 release. How about rebasing this into 2.0 branch?

Reply:
OK, I rebased to 2.0 (I think)